webapp/platform/client/src/client4.ts

```diff
@@ -3283,6 +3287,14 @@ export default class Client4 {
         );
     };
 
+    // AI Routes
+    getAIAgents = () => {
+        return this.doFetch<{agents: Array<{id: string; displayName: string; username: string; service_id: string; service_type: string}>}>(
```

Comment:
Nit: this is long enough to have its own type somewhere. We can create a new type file for this I would think